server/qa/lib/qa.ex

```diff
@@ -0,0 +1,50 @@
+defmodule QA do
+  @moduledoc """
+  This module is a CLI for running QA tests on macOS.
+  """
+  use Application
+
+  alias QA.Agent
+
+  def start(_, _) do
+    params = parse_arguments()
+
+    opts = [
+      anthropic_api_key: params.anthropic_api_key
+    ]
+
+    case Agent.test(params, opts) do
+      :ok ->
+        IO.puts("Finished running QA test")
```

Comment:
Let's use `Logger` here.

Reply:
I think for a direct CLI output, `IO.puts` is a better choice. `Logger` should be reserved to logging, not a cli `stdout`